projects/packages/forms/changelog/update-forms-block-rename-creativemail

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: changed
+
+Forms block: rename CreativeMail integration from "Newsletter" to "CreativeMail" to avoid confusing with "Jetpack Newsletters" and subscription block.
```

Comment:
```suggestion
Forms block: rename "Newsletter Connection" to "Creative Mail" to avoid confusing with "Jetpack Newsletters" and subscription block. Call the block a "Lead Capture" block (not sign up).
```